product-scenarios/1-integrating-systems-that-communicate-in-heterogeneous-message-formats/1.1-converting-soap-to-json/1.1.1-soap-to-json-using-payloadfactory-mediator/src/test/java/org/wso2/carbon/esb/endpoint/test/ScenarioTestBase.java

```diff
@@ -0,0 +1,85 @@
+/*
+ *Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *WSO2 Inc. licenses this file to you under the Apache License,
+ *Version 2.0 (the "License"); you may not use this file except
+ *in compliance with the License.
+ *You may obtain a copy of the License at
+ *
+ *http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *Unless required by applicable law or agreed to in writing,
+ *software distributed under the License is distributed on an
+ *"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *KIND, either express or implied.  See the License for the
+ *specific language governing permissions and limitations
+ *under the License.
+ */
+
+package org.wso2.carbon.esb.endpoint.test;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Properties;
+
+public class ScenarioTestBase {
+
+    private static final String INPUTS_LOCATION = System.getenv("DATA_BUCKET_LOCATION");
+    private static final String INFRASTRUCTURE_PROPERTIES = "infrastructure.properties";
+    private static final String DEPLOYMENT_PROPERTIES = "deployment.properties";
+    private static final String JOB_PROPERTIES = "testplan-props.properties";
+
+    private static final Log log = LogFactory.getLog(ScenarioTestBase.class);
+
+    public static final String MGT_CONSOLE_URL = "MgtConsoleUrl";
+    public static final String CARBON_SERVER_URL = "CarbonServerUrl";
+    public static final String ESB_HTTP_URL = "ESBHttpUrl";
+    public static final String ESB_HTTPS_URL = "ESBHttpsUrl";
+
+    protected static final int ARTIFACT_DEPLOYMENT_WAIT_TIME_MS = 120000;
+    protected static final String resourceLocation = System.getProperty("test.resources.dir");
+
+    /**
+     * This is a utility method to load the deployment details.
+     * The deployment details are available as key-value pairs in {@link #INFRASTRUCTURE_PROPERTIES},
+     * {@link #DEPLOYMENT_PROPERTIES}, and {@link #JOB_PROPERTIES} under the
+     * {@link #INPUTS_LOCATION}.
+     *
```

Comment:
Please remove the additional line.

Reply:
This is fixed by PR [https://github.com/wso2/product-ei/pull/3471](https://github.com/wso2/product-ei/pull/3471)